themes/default/content/pulumi-up/_index.md

```diff
@@ -2,7 +2,7 @@
 title: PulumiUP
 type: page
 layout: pulumi-up-2022
-meta_image: /images/pulumiup/save-the-date-pulumiup.png
+meta_image: /images/pulumiup/2022-save-the-date.png
 meta_desc: |
     PulumiUP introduces you to the emerging practice of Cloud Engineering. Hear from the Pulumi team
     and assorted industry experts.
```

Comment:
```suggestion
meta_desc: |
    Join us for PulumiUP 2022: free virtual conference on May 4th. Let's talk about the future of IaC, Cloud Engineering & #DevOps and Cloud. Hear from industry-recognized leaders, attend technical talks, demos, and panel discussions. #PulumiUP #infrastructureascode
```

Reply:
@SaraDPH we have some validation that only allows meta descriptions to be at most 160 characters. We don't currently support separate text for social cards, but it is something we can implement!

Reply:
Definitely support adding this! Would be good to submit it as a follow-up PR, though -- I'll make an issue for that and add it to the list. (Should be a quick follow-up to this one.)